runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -39,18 +39,11 @@
 @Slf4j
 public class ObjectsView extends AbstractView {
 
-    /**
-     * The Transaction stream is used to log/write successful transactions from different clients.
-     * Transaction data and meta data can be obtained by reading this stream.
-     */
-    @Deprecated
-    public static final UUID TRANSACTION_STREAM_ID = CorfuRuntime.getStreamID("Transaction_Stream");
+    private static final String LOG_REPLICATOR_STREAM_NAME = "Transaction_Stream";
```

Comment:
let's change the name of this stream to be suggestive to LR, perhaps "LR_Transaction_Stream"